From 44636a7c5862538def0d2fac074772cc39a22a15 Mon Sep 17 00:00:00 2001
From: Rudi Heitbaum <rudi@heitbaum.com>
Date: Tue, 16 May 2023 02:27:59 +1000
Subject: [PATCH] alsactl: fix compilation when building in a subdir

Fixes: 613372d
Fixes: cff2d1c

Compile errors when building in a subdir:
alsactl/alsactl.c:33:10: fatal error: os_compat.h: No such file or directory
   33 | #include "os_compat.h"
      |          ^~~~~~~~~~~~~
alsactl/lock.c:34:10: fatal error: os_compat.h: No such file or directory
   34 | #include "os_compat.h"
      |          ^~~~~~~~~~~~~

Signed-off-by: Rudi Heitbaum <rudi@heitbaum.com>
---
 alsactl/Makefile.am | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/alsactl/Makefile.am b/alsactl/Makefile.am
index 80dba69d..fb32c4f3 100644
--- a/alsactl/Makefile.am
+++ b/alsactl/Makefile.am
@@ -9,6 +9,8 @@ EXTRA_DIST=alsactl.1 alsactl_init.xml
 
 AM_CFLAGS = -D_GNU_SOURCE
 
+AM_CPPFLAGS = -I$(top_srcdir)/include
+
 alsactl_SOURCES=alsactl.c state.c lock.c utils.c init_parse.c init_ucm.c \
 		daemon.c monitor.c clean.c info.c
 
